packaging/debian/watch

```diff
@@ -0,0 +1,2 @@
+version=3
+https://www.gen2vdr.de/wirbel/w_scan/w_scan-(.*).tar.bz2 debian /bin/sh debian/repack-upstream.sh
```

Comment:
This should be also changed. Probably to "https://github.com/stefantalpalaru/w_scan2/archive/(.*).tar.gz"